src/components/atoms/NewsFlahsFilterPanel.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ynetLogo from  '../../assets/Ynet_website_logo.svg';
+import wallaLogo from  '../../assets/walla-logo.svg';
+import {Box, makeStyles} from '@material-ui/core';
+import Button from "@material-ui/core/Button";
```

Comment:
Please move 3rd party imports to the top of the list, before imports of our own components.